pages/Javascript-API.md

```diff
@@ -121,6 +112,43 @@ Re-parses the `scene.config` object and redraws the scene, updating data sources
 scene.updateConfig()
 ```
 
+## Events
+
+Tangram provides two selection event handlers: `hover` and `click`, and one render state event emitter: `view_complete`.
```

Comment:
We also have events for:

- `load`: scene was loaded, event callback is passed an object with a `config` property, containing the just-loaded scene config object
- `warning`: an issue occurred while processing the scene, the callback is passed an object with a `type`, which indicates the scope of the issue (e.g. `textures`, `sources`, etc.), along with additional type-specific properties
- `error`: an unrecoverable error occurred while processing the scene, the callback is passed an object with `type`, `message` (text error message), `error` (JS error object), and `url` (the URL from which the scene was loaded) properties

We should probably document these as well, though `warning` and `error` are most applicable to Tangram Play right now.